src/Storages/MergeTree/MutateTask.cpp

```diff
@@ -1351,6 +1355,198 @@ class MutateSomePartColumnsTask : public IExecutableTask
     std::unique_ptr<PartMergerWriter> part_merger_writer_task{nullptr};
 };
 
+class LightweightDeleteTask : public IExecutableTask
+{
+public:
+
+    explicit LightweightDeleteTask(MutationContextPtr ctx_) : ctx(ctx_) {}
+
+    void onCompleted() override { throw Exception(ErrorCodes::LOGICAL_ERROR, "Not implemented"); }
```

Comment:
why throw exceptions in these methods?

Reply:
Referenced from MutateSomePartColumnsTask, not sure why.